src/locale/locales/ca/messages.po

```diff
@@ -3856,12 +3855,12 @@ msgid "users followed by <0/>"
 msgstr "usuaris seguits per <0/>"
 
 #: src/view/com/modals/Threadgate.tsx:106
-msgid "Users in \"{0}\""
+msgid "Users in \"{0}\
```

Comment:
Missing quotation marks here I think? Not sure if it matters

Reply:
Thank you, you are really helpful

Reply:
Happy to help :)